### PR TITLE
Fix/incoming track status

### PR DESCRIPTION
**Pull request by HoseinHaqiqian**
It fixes incoming track status `recvonly/sendonly` on sdp offers request.

### CODE REVIEW

**Review by kasemj — COMMENTED** [review]
Other than the one comment i have everything looks good to me.

File: lib/src/room.dart [hunk 1]
```diff
@@ -561,7 +562,10 @@ class KurentoRoom extends ChangeNotifier implements Room {
         break;
 
       case ParticipantMessageType.INCOMING_TRACK_CREATE:
-        await _createIncomingTrack(participantMessage.trackId);
+        final isAudio = bool.parse(participantMessage.payload['audio'].toString());
```
[the remark this answers is not in the document]
Comment (HoseinHaqiqian) [reply lib/src/room.dart]: Resolved by using tryParse in participantMessage model

**Review by kasemj — DISMISSED** [review]
LGTM

**Comment by kasemj** [comment]
@HoseinHaqiqian Oh one thing before I merge. Can you update `CHANGELOG.md` and `pubspec.yaml`

File: lib/src/models/track.dart [hunk 2]
```diff
@@ -2,11 +2,15 @@ class Track {
...
   int participantId;
   int? incomingTrackId;
+  TrackKind? kind;
 
   Track.fromJson(Map<String, dynamic> json)
       : id = int.parse(json['id']),
         participantId = json['participantId'],
-        incomingTrackId = int.tryParse(json['incomingTrackId'] ?? '');
+        incomingTrackId = int.tryParse(json['incomingTrackId'] ?? ''),
+        kind = json['audio'] ?? false ? TrackKind.audio : (json['video'] ?? false ? TrackKind.video : null);
```
Comment (bwang-aira) [inline lib/src/models/track.dart:11]: This line could benefit from adding a comment. Also, this line seems longer than 80 chars, please make sure your IDE is set up to limit each line to 80 chars.

**Pull request merged**